src/include/iscsi-lio-server/helps.rb

```diff
@@ -155,6 +161,9 @@ def initialize_iscsi_lio_server_helps(include_target)
             "<p>Under <b>Ip Address</b> and <b>Port Number</b> you specify under which address\n" +
               "and port the service will be available. Default for port number is 3260.\n" +
               "Only ip addresses assigned to one of the network cards are possible."
+          ) + "</p>" +"<p>" +
+          _(
```

Comment:
this won't work. you need to translate it above. another way is to mark string for translation with `N_()` and then use `_()` here.